kratos/utilities/delaunator_utilities.cpp

```diff
@@ -0,0 +1,206 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+#define REAL double
+
+#if !defined(KRATOS_TRIANGLE_EXTERNAL_H_INCLUDED)
+#define  KRATOS_TRIANGLE_EXTERNAL_H_INCLUDED
```

Comment:
i would include the include ward in the triangle.h file (maybe #include once)

Reply:
OK

Reply:
I just checked and this is exactly the same as using #include once, in fact #include once is not supported by all compilers

Reply:
the point is that the guard does not have to be added every time triangle is included

Reply:
so once you did this I think this is ready to go in

Reply:
What I am supposed to do?, this is the gaurd we use in all files

Reply:
yes, the point is that we DO NOT WANT this guard in ALL THE FILES but instead only once in [triangle.h](https://github.com/KratosMultiphysics/Kratos/blob/master/external_libraries/triangle/triangle.h)

Reply:
OK, I am not very pro modify external libraries, but as triangle has been the same since forever I think we can live with that

Reply:
that was also our thinking